src/posts/how-to-use-JAWS-on-Windows

```diff
@@ -0,0 +1,47 @@
+JAWS, Job Access With Speech, is a screen reader for Windows operating system users. JAWS assists individuals with visual impairments with interacting with computers by helping them navigate applications, read documents and browse the web on Windows operating systems. 
+
+In this article, we will provide step-by-step instructions on how to start using JAWS on Windows.
+
+## Install JAWS
+First, the user needs to install JAWS in order to use it. JAWS have several subscription plans for their users. JAWS can be installed on their website, [Freedom Scientific website](https://www.freedomscientific.com/products/software/jaws/). You can purchase a valid license that fits you. After purchasing a license, you can find JAWS on your download file or a file you saved the program in. Double click on it to start the installation process. The installer will guide you through the installation process, which will help you choose your desired language and installation options. After successfully downloading JAWS, restart your computer to finalize the set-up. You now have JAWS and are ready to use it.
+
+## Getting Started with JAWS, (Basic commands)
+Once JAWS is installed and you have restarted your computer, you are ready to use JAWS! You can manually launch JAWS by double clicking on the JAWS icon on your desktop or selecting it from your Start menu. After running JAWS, you can use your keyboard shortcuts to interact with your computer. Below are some basic commands you can familiarize yourself with to get started using JAWS.
+- INSERT + H: This commands opens the JAWS Help system. The JAWS Help system provides comprehensive documentation on how to use JAWS effectively.
+- Arrow Keys (Basic Navigation): JAWS relies on keyboard commands for navigation. Arrow keys allow you to move between elements on the screen, such as links, heading or form fields.
+- TAB Key: Pressing TAB Key allows you to move between interactive elements in the application. 
+- H: Pressing the H key moved you between headings.
+- INSERT + DOWN ARROW: This keyboard shortcut will allow you to read the current line in the website.
+- INSERT + UP ARROW: This keyboard shortcut will allow you to read from the current position to the top of the document or application.
+- ALT + TAB: This keyboard shortcut will allow you to switch between open applications or windows.
+- WINDOWS KEY + M: This will minimize all windows, allowing you to access the desktop quickly.
+- Control + F: This key allows you to open the search feature within your browser.
+Listed here are some basic keyboard shortcuts you can use to start using JAWS right away. For a full list of shortcut commands, you can visit [JAWS website](https://doccenter.freedomscientific.com/doccenter/doccenter/rs25c51746a0cc/2011-10-04_jawsandmagicreadingcommands/JAWSKeystrokes.htm). JAWS provide many keyboard shortcuts for a variety of reasons. In the next section, we will explore some advanced features JAWS provide to their users. 
+
```

Comment:
Because our audience is primarily web developers, are there any control scenarios specific to web developers we might want to cover? 

- user enter/space with interactive elements?
- how to navigate tables?
- how to open/close a select box?
- how to open/close and navigate a native date-picker?

I like the `H` key for headings. I like the `TAB` key for interactive elements. But is there anything else I need to successfully validate a website. Could be another `## JAWS for web developers` section?

Reply:
@davatron5000 I added a new `JAWS for web developer` section. Thank you for suggesting.